source/views/menus/dev-bonapp-picker.js

```diff
@@ -1,73 +0,0 @@
-// @flow
```

Comment:
Why?

Reply:
Wasn't being used anywhere...should we keep it?